ush/forecast_postdet.sh

```diff
@@ -354,8 +354,8 @@ FV3_out() {
     for restart_date in "${restart_dates[@]}"; do
       echo "Copying FV3 restarts for 'RUN=${RUN}' at ${restart_date}"
       for fv3_file in ${file_list}; do
-        ${NCP} "${DATArestart}/FV3_RESTART/${restart_date}.${fv3_file}" \
-               "${COMOUT_ATMOS_RESTART}/${restart_date}.${fv3_file}"
+        cpreq "${DATArestart}/FV3_RESTART/${restart_date}.${fv3_file}" \
```

Comment:
Copy is going to a COM:
```suggestion
        cpfs "${DATArestart}/FV3_RESTART/${restart_date}.${fv3_file}" \
```